kuzzleio/documentation-V2
# Refactor Index controller documentation with the new structure

## What does this PR do?

Refactor Index controller documentation with the new structure

### How should this be manually tested?

https://deploy-preview-23--kuzzle-doc-v2.netlify.com/sdk-reference/index/create/
check all `Index` controller methods in each language

### Other changes
* Improve scaffolding templates
* Enhance CSS for table and inline hilighting
